src/dreyfus_fabric_cleanup.erl

```diff
@@ -26,10 +26,20 @@ go(DbName) ->
     {ok, DesignDocs} = fabric:design_docs(DbName),
     ActiveSigs = lists:usort(lists:flatmap(fun active_sigs/1,
         [couch_doc:from_json_obj(DD) || DD <- DesignDocs])),
+    cleanup_local_doc(DbName, ActiveSigs),
     clouseau_rpc:cleanup(DbName, ActiveSigs),
+    ok.
+
+active_sigs(#doc{body={Fields}}=Doc) ->
+    {RawIndexes} = couch_util:get_value(<<"indexes">>, Fields, {[]}),
+    {IndexNames, _} = lists:unzip(RawIndexes),
+    [begin
+         {ok, Index} = dreyfus_index:design_doc_to_index(Doc, IndexName),
+         Index#index.sig
+     end || IndexName <- IndexNames].
 
-    %clean local doc
-    BaseDir = config:get("couchdb", "search_index_dir", "/srv/search_index"),
+cleanup_local_doc(DbName, ActiveSigs) ->
+    {ok, BaseDir} = clouseau_rpc:get_root_dir(),
```

Comment:
Hi Mayya. 
a) I checked the config file and there is no record of `search_index_dir`.
b) The search index dir is controlled by `clouseau` in fact, so I think it is better to get `BaseDir` from `clouseau`.